.github/workflows/config.yml

```
@@ -0,0 +1,51 @@
+name: config
+
+on:
+  push:
+    branches:
+      - master
+      - test
```

```suggestion
on: [push, pull_request]
```

we can specify both `push` and `pull_request` to capture both cases

---

yaa ok i will add for pullrequest to for main branch only right pull requests